.depcheckrc.yml

```diff
@@ -23,3 +23,4 @@ ignores:
   - 'babel-jest'
   # Ignore dependencies which plug into the NPM lifecycle
   - '@lavamoat/preinstall-always-fail'
+  - 'react-native'
```

Comment:
Let's create an issue to eventually move react-native as a dependency for the root monorpo if tamagui was able to do it we should too